src/resumes/purple.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+<div class="resume" id="template">
+    <div id="resume-header">
+        <div id="header-left">
+            <h2 id="position">{{person.position}}</h2>
+            <h1 id="name">{{person.name.first}} {{person.name.last}}</h1>
+            <div id="info-flex">
+                <span id="email"><a :href='person.contact.mailto'><i class="fa fa-envelope" aria-hidden="true"></i> {{person.contact.email}}</a></span>
+                <span id="phone"><i class='fa fa-phone-square' aria-hidden="true"></i> {{person.contact.phone}}</span>
+                <span id="website"><a :href='person.contact.website'><i class="fa fa-home" aria-hidden="true"></i> {{person.contact.website}}</a></span>
+                <span id="github"><a :href='person.contact.githubFull'><i class="fa fa-github" aria-hidden="true"></i> {{person.contact.github}}</a></span>
+            </div>
+        </div>
+        <div id="header-right">
+            <div id="headshot"></div>
+        </div>
+    </div>
+    <div id="resume-body">
+        <div id="experience-container">
+            <h2 id="experience-title">EXPERIENCE</h2>
+            <div class="spacer"></div>
+            <div class="experience" v-for="experience in person.experience">
+                <h2 class="company">{{experience.company}}</h2>
+                <p class="job-info"><span class="job-title">{{experience.position}} | </span><span class="experience-timeperiod">{{experience.timeperiod}}</span></p>
+                <p class="job-description" v-if="experience.description">{{experience.description}}</p>
+                <ul v-if="experience.list" > 
+                    <li v-for="item in experience.list"><span class="list-item-black">{{item}}</span></li>
+                </ul>
+            </div>
+        </div>
+        <div id="education-container">
+            <h2>EDUCATION</h2>
+            <div class="spacer"></div>
+            <div class="education" v-for="education in person.education">
+                <h2 class="education-description">{{education.description}}</h2>
+                <p><span class="degree">{{education.degree}} | </span><span class="education-timeperiod">{{education.timeperiod}}</span></p>
+            </div>
+        </div>
+        <div id="skills-container" v-if="person.skills != []">
+            <h2>SKILLS</h2>
+            <div class="spacer"></div>
+            <p id="skill-description">{{person.skillDescription}}</p>
+            <ul id="skill-list">
+                <li class="skill" v-for="skill in person.skills"><span class="list-item-black">{{skill.name}}</span></li>
+            </ul>
+        </div>
+    </div>
+    <div id="resume-footer">
+        <div v-if="person.about">
+            <h2>About Me</h2>
+            <p>{{person.about}}</p>
+        </div>
+    </div>
+</div>
+</template>
+
+<script>
+import {
+  PERSON
+} from '../person';
+
+import Vue from 'vue';
+
+PERSON.contact.githubFull = 'https://github.com/' + PERSON.contact.github;
```

Comment:
I'm not sure if mutating the person-object is a good idea. If more than one template does this, we have a problem in the web-preview because its all just one single vue-instance with one shared person-object.

Reply:
I didn't think about that - good call. I'll change it.